docs/locale/fr/LC_MESSAGES/userguide/recovery_device.po

```diff
@@ -0,0 +1,207 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2016-present, Scille SAS
+# This file is distributed under the same license as the Parsec package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2023.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: Parsec 2.16.0-a.0+dev\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-11-10 09:43+0000\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language: fr\n"
+"Language-Team: fr <LL@li.org>\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.13.0\n"
+
+#: ../../userguide/recovery_device.rst:7
+msgid "Recover devices"
+msgstr "Récupérer des appareils"
+
+#: ../../userguide/recovery_device.rst:9
+msgid ""
+"A *recovery device* is a backup of an existing device. If you lose access to "
+"Parsec (you forget the password), the recovery device will allow you to "
+"recreate a new device and set a new password."
+msgstr ""
+"Un *appareil de récupération* est une sauvegarde d'un appareil existant. Si "
+"vous perdez accès à Parsec (vous avez oublié votre mots de passe), "
+"L'appareil de récupération vous permet de recréer un nouvel appareil avec un "
+"nouveau mot de passe."
+
+#: ../../userguide/recovery_device.rst:11
+msgid ""
+"In order to `Create a new recovery device`_ or `Use a recovery device`_ you "
+"need to open the recovery device dialog from the main menu:"
+msgstr ""
+"Pour pouvoir `Créer un nouvel appareil de récupération`_ ou d'`Utiliser un "
+"appareil de récupération`_ vous avez besoin d'ouvrir le dialogue de "
+"récupération d'appareil dans le menu principal:"
+
+#: ../../userguide/recovery_device.rst:13
+msgid "Open the ``Recovery device`` menu."
+msgstr "Ouvrir le menu ``Récupération d'appareils``."
+
+#: ../../userguide/recovery_device.rst:15
+msgid ""
+"You need to open the sub-menu then click on ``Recovery device`` option "
+"(alternatively, you can use the shortcut ``Ctrl+I``)."
+msgstr ""
+"Vous avez besoin d'ouvrir le sous-menu puis cliquer sur ``récupération "
+"d'appareil`` (Alternativement, vous pouvez utiliser le raccourci ``Ctrl+I``)."
+
+#: ../../userguide/recovery_device.rst
+#, fuzzy
+msgid "Parsec menu with recovery device option highlighted"
+msgstr "Le menu Parsec avec l'option récupération d'appareil surligné"
+
+#: ../../userguide/recovery_device.rst:20
+msgid "That will open a modal dialog that looks like this:"
+msgstr "Ça va ouvrir un dialogue qui ressemble à ceci:"
```

Comment:
```suggestion
msgstr "La boîte de dialogue suivant apparaît alors:"
```

Reply:
Why `dialogue` need to be changed ?

Reply:
I think that "dialogue" and "modal" are not that used alone in French. It should be "[boîte de dialogue]"(https://fr.wikipedia.org/wiki/Bo%C3%AEte_de_dialogue).

Reply:
done